es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/CreateEsHttpRequest.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ *     Beshu Limited all rights reserved
+ */
+package tech.beshu.ror.proxy.es
+
+import java.util
+
+import com.twitter.finagle.http.{Method, Request, Version}
+import io.netty.handler.codec.http.HttpHeaderNames
+import io.netty.handler.codec.http.cookie.{ServerCookieDecoder, ServerCookieEncoder}
+import org.elasticsearch.common.bytes.{BytesArray, BytesReference}
+import org.elasticsearch.rest.{RestRequest, RestStatus}
+import org.elasticsearch.http.{HttpRequest => EsHttpRequest, HttpResponse => EsHttpResponse}
+import scala.collection.JavaConverters._
+
+object CreateEsHttpRequest {
+
+  def from(request: Request): EsHttpRequest = from(
+    request,
+    request
+      .headerMap
+      .keys
+      .map(key => (key, request.headerMap.getAll(key).toList))
+      .toMap
+  )
+
+  private def from(request: Request,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)